unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/NftBreadcrumbComponentView.cs

```diff
@@ -34,6 +39,69 @@ public override void Awake()
         }
 
         public override void RefreshControl()
+        {
+            ClearInstances();
+
+            categoriesByIndex = new NftSubCategoryFilterComponentView[model.Path.Length];
```

Comment:
I could not find any usages of this array besides creating and assigning

Reply:
Done